graphics/graphics_icons.xml

```diff
@@ -8,7 +8,7 @@
 
 	<!-- CoreRPG -->
 	<!-- Portrait -->
-	<icon name="charlist_base" file="graphics/pngs/portrait.png" />
+	<icon name="charlist_base" file="graphics/pngs/portrait_token.png" />
```

Comment:
Fix for the missing gold frame around PC portraits (the left one in 5e... right one was always ok).